docs/user/sbt.rst

```diff
@@ -9,7 +9,7 @@ Start within a new folder, and create a file ``project/plugins.sbt`` as follows:
 
     resolvers += Resolver.sonatypeRepo("snapshots")
 
-    addSbtPlugin("org.scala-native" % "sbtplugin"  % "0.1-SNAPSHOT")
+    addSbtPlugin("org.scala-native" % "sbtplugin"  % "0.1.0-SNAPSHOT")
```

Comment:
hum there is something strange in the sbt build. it does not compile if I set the version to 0.1.0-SNAPSHOT. _If it's not broken dont fix it_